server/src/web/auth.controller.spec.ts

```diff
@@ -105,14 +105,14 @@ describe('AuthController', () => {
       expect(await controller.signIn(signInRequestDto)).toBe(authDto);
     });
 
-    it('should throw "user not found"', async () => {
+    it('should return null', async () => {
       when(mockAuthService.signUp(address, name, message, signature))
         .thenReject().thenThrow(new UnauthorizedException());
 
       controller = new AuthController(instance(mockAuthService));
 
-      expect(await controller.createToken(createTokenRequestDto))
-        .toEqual(undefined);
+      expect(await controller.signIn(signInRequestDto))
```

Comment:
signIn is like 

```
async signIn(address: string, message: string, signature: string): Promise<SignInResponseDto> {
        const result: boolean = this.validationService.verify(address, message, signature);
        if (!result) {
            throw new UnauthorizedException();
        }

        const user: User = await this.userService.getByAddress(address);
        if (user === undefined) {
            throw new NotFoundException();
```

i can not understand how can it return null when error is thrown.

Reply:
@inDlife It was null when I checked the return value after UnauthorizedException().